main_test.go

```diff
@@ -183,3 +183,33 @@ func testCreateConfFile(dirs []string, resolver, deployer string, t *testing.T,
 	}
 	return nil
 }
+
+func runNewCli(t *testing.T, args ...string) {
```

Comment:
runNewCli --> runCli
(not sure that the *New* has a real meaning here).

Reply:
Done.